superset-frontend/src/common/components/Collapse/index.tsx

```diff
@@ -18,11 +18,10 @@
  */
 import React from 'react';
 import { styled } from '@superset-ui/core';
-// eslint-disable-next-line no-restricted-imports
-import { Collapse as AntdCollapse } from 'antd';
+import { Collapse as AntdCollapse } from 'src/common/components';
```

Comment:
I feel like our patterns have become a bit scattered, but if we want to encourage the use of THIS `Collapse` component, then we should probably import the AntD one directly from `antd` here, and NOT export the AntD `Collapse` component in `common/components/index.tsx`.

Reply:
I totally agree. This happened because of the ESLint rule `no-restricted-imports` that prevented AntD imports and recommended imports from `src/common/components`. To fix this, I created two `.eslintrc.js` files in `src/common/components` and `src/components` disabling this rule for these specific packages. I also removed the AntD exports in `src/common/components` for some of the components that I worked on.